src/main/proto/oauth2.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.ea.restaurant.grpc";
+
+message LoginClientResponse{
```

Comment:
Maybe you can convert this to a simple java class

Reply:
Done.